falcon/media/msgpack.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+
+from falcon import errors
+from falcon.media import BaseHandler
+
+
+class MessagePackHandler(BaseHandler):
+    """Handler built using the :py:mod:`msgpack` module from python-msgpack
+
+    Requires the ``python-msgpack`` module to be installed.
```

Comment:
[suggestion] This probably deserves to be highlighted as a `Note`. Also, `python-msgpack` is technically a package, not a module. 😈